File: POO JAVA/Aula10/src/aula10/Pessoa.java
package aula10;

public class Pessoa {

    //Atributos da SuperClasse
    private String nome;
    private int idade;
    private String sexo;
    
    //Metodos Acessores e Modificadores
    public String getNome(){    
        return nome;    
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    public int getIdade() {
        return idade;
    }
    public void setIdade(int idade) {
        this.idade = idade;
    }
    public String getSexo() {
        return sexo;
    }
    public void setSexo(String sexo) {    
        this.sexo = sexo;
    }

    //Metodos da Classe
    public void fazerAniversario() {
        this.idade++;
    }

    @Override
    public String toString() {
        return "Pessoa{" + "nome=" + nome + ", idade=" + idade + ", sexo=" + sexo + '}';
    }
    
}

File: POO PHP/Aula10/Pessoa.php
<?php

class Pessoa {

    //Atributos da Classe
    protected $nome;
    private $idade;
    private $sexo;
    
    //Metodos Acessores e Modificadores
    public function getNome() {
        return $this->nome;
    }
    public function getIdade() {
        return $this->idade;
    }
    public function getSexo() {
        return $this->sexo;
    }
    public function setNome($nome): void {
        $this->nome = $nome;
    }
    public function setIdade($idade): void {
        $this->idade = $idade;
    }
    public function setSexo($sexo): void {
        $this->sexo = $sexo;
    }
    
    //Metodo da Classe
    public function fazerAniversario() {
        $this->idade++;
    }
}
